### DISCUSSION TITLE
How to create mcp client that connect to mcp servers ? Claude is 1 client to multiple server (I Think)

### DISCUSSION THREAD

**Discussion by x4080** [modelcontextprotocol/modelcontextprotocol#1252] [Q&A - Client implementation]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [ ] Server Implementation
- [ ] General Implementation
- [ ] Documentation
- [ ] Other

### Your Question

How to create mcp client that connect to mcp servers ? Claude is for example, is there any example in python ?

Thanks

**Comment by headinthebox** [comment]
see here: https://github.com/orgs/modelcontextprotocol/discussions/23

**Comment by localden** [comment]
We've limited Discussions in this repository to meeting notes and are closing this thread. To continue the conversation, pick a channel from [Contributor Communication](https://modelcontextprotocol.io/community/communication). Thanks for posting.
